src/status_im/contexts/preview/feature_flags/view.cljs

```diff
@@ -16,24 +16,27 @@
      :title      "Features Flags"
      :icon-name  :i/arrow-left
      :on-press   #(rf/dispatch [:navigate-back])}]
-   (doall
-    (for [context-name ff/feature-flags-categories
-          :let         [context-flags (filter (fn [[k]]
-                                                (string/includes? (str k) context-name))
-                                              (ff/feature-flags))]]
-      ^{:key (str context-name)}
-      [rn/view {:style style/container}
-       [quo/text
-        context-name]
-       (doall
-        (for [i    (range (count context-flags))
-              :let [[flag] (nth context-flags i)]]
-          ^{:key (str context-name flag i)}
-          [rn/view {:style {:flex-direction :row}}
-           [quo/selectors
-            {:type            :toggle
-             :checked?        (ff/enabled? flag)
-             :container-style {:margin-right 8}
-             :on-change       #(ff/toggle flag)}]
-           [quo/text (second (string/split (name flag) "."))]]))]))])
+   [rn/scroll-view
+    (doall
+     (for [context-name ff/feature-flags-categories
+           :let         [context-flags (filter (fn [[k]]
+                                                 (= context-name
+                                                    (first (string/split (str (name k)) "."))))
+                                               (ff/feature-flags))]]
+       ^{:key (str context-name)}
+       [rn/view {:style style/container}
+        [quo/text
+         context-name]
+        (doall
+         (for [i    (range (count context-flags))
+               :let [[flag] (nth context-flags i)]]
+           ^{:key (str context-name flag i)}
+           [rn/view {:style {:flex-direction :row}}
+            [quo/selectors
+             {:type            :toggle
+              :checked?        (ff/enabled? flag)
+              :container-style {:margin-right 8}
+              :on-change       #(ff/toggle flag)}]
+            [quo/text (second (string/split (name flag) "."))]]))]))]
+  ])
```

Comment:
Swinging loosely

Reply:
Yeah I feel like I'm gambling with strings right now 😄 
I also think we should come up with a way to make the feature flags a little more robust. For example, we could allow for multiple levels of namespacing e.g. `wallet.settings.stuff`. But I've decided to wait for another PR 😅